backend/app/models/inquiry.py

```diff
@@ -1,27 +1,33 @@
 import uuid
 from datetime import datetime, timezone
+from typing import List, Optional
 
-from sqlmodel import Field, SQLModel
+from sqlmodel import Field, Relationship, SQLModel
+
+
+class Response(SQLModel, table=True):
+    id: uuid.UUID = Field(default_factory=uuid.uuid4, primary_key=True)
+    content: str = Field(min_length=1, max_length=255)
+    inquiry_id: uuid.UUID = Field(foreign_key="inquiry.id")
+    inquiry: Optional["Inquiry"] = Relationship(back_populates="responses")
 
 
 # Shared properties
 class InquiryBase(SQLModel):
     text: str = Field(min_length=10, max_length=255)
 
 
-# Properties to receive on inquiry creation
 class InquiryCreate(InquiryBase):
     text: str = Field(min_length=10, max_length=255)
 
 
-# Database model, database table inferred from class name
 class Inquiry(InquiryBase, table=True):
     id: uuid.UUID = Field(default_factory=uuid.uuid4, primary_key=True)
     text: str = Field(min_length=10, max_length=255, unique=True)
     created_at: datetime = Field(default_factory=lambda: datetime.now(timezone.utc))
+    responses: List[Response] = Relationship(back_populates="inquiry")
```

Comment:
There should not be responses on the inquiry model. An Inquiry doesn't know anything about the response. Instead, the Response model will have a reference the inquiry. Checkout the [entity-relationship-diagram](https://lucid.app/lucidchart/2d988dda-8c81-4024-9cfe-33e23521288b/edit?viewport_loc=1965%2C-1160%2C2705%2C1598%2C0_0&invitationId=inv_32e97de0-7f01-43aa-85d3-72ad0b6e1f2b) that we've been working on.

Reply:
Thanks

Reply:
fixed I think